torchao/quantization/fp6_llm.py

```diff
@@ -111,6 +112,143 @@ def from_tc_float6_e3m2(tensor: Tensor, M: int, N: int, dtype: torch.dtype = tor
     return from_float6_e3m2(tensor_fp6, no_bit_packing=True, dtype=dtype)
 
 
+# https://github.com/microsoft/DeepSpeed/blob/3a3a6db3332e339cc9fd94efd4982f6d60635a3d/deepspeed/inference/v2/kernels/core_ops/cuda_linear/cuda_linear.py
+_SPLIT_K_MAP = [
+    {  # tokens: [1, 64]
```

Comment:
n00b q: what is meant by token counts here?

Also can these values be autotuned? I don't necesarily wanna merge in something that's only fast on A100

Reply:
I think it's the batch size. Yea, it will be great if we can autotune CUDA kernel also.